unison-src/transcripts/idempotent/fix-ls.md

```diff
@@ -26,10 +25,10 @@ foo.bar.subtract x y = x Int.- y
 ``` ucm
 test-ls/main> add
 
-  ⍟ I've added these definitions:
+  Okay, I'm searching the branch for code that needs to be
+  updated...
 
-    foo.bar.add      : Int -> Int -> Int
-    foo.bar.subtract : Int -> Int -> Int
+  Done.
```

Comment:
I notice `add` used to show what actually changed; is that something we should have in `update` too?

Reply:
We certainly could!

Reply:
We should! We can update the display template design and take a look after any already in progress `merge`/`update` stuff